src/domain/chains/entities/__tests__/block-explorer-uri-template.factory.ts

```diff
@@ -1,3 +1,5 @@
+/* istanbul ignore file */
+
```

Comment:
For now, I added these to the the factories of chains. We can think of a rule that would match any file under `__tests__` and that ends with `.factory.ts`.

I don't think the factories should be included in the coverage as they serve more of a utility for the tests. The branches that were being evaluated in the coverage were specific to having an argument provided or not (nullability check `??`).

Reply:
Proposal under https://github.com/5afe/safe-client-gateway-nest/pull/94

Reply:
Reverted in [d6ed2b7](https://github.com/5afe/safe-client-gateway-nest/pull/93/commits/d6ed2b745d8c6032e314033ff3b5192978e8f60a) because #94 was merged